README.md

```diff
@@ -15,4 +15,5 @@ ruby library or CVS via cvs specialized library.
 
```

Comment:
The first layer provides access to the file and its content. By default it accesses the local system, but it can be replaced by alternatives to work on a chroot environment, on test data, on a remote machine or any other scenario.

The second layer is the parser, that understands the structure of the configuration files. There are also several possible variants like...